index.py

```diff
@@ -0,0 +1,93 @@
+def sayiKontrol (value, type = None):
+    if (not(value.isdigit())):
+        return sayiKontrol(input("Tekrar giriniz:"), type)    
+
+    value = int(value)
+
+    if (
+        (type == "harfNotu" and value >= 0 and value <= 100) or 
+        (type == "menu" and value > 0 and value < 4) or
+        type == None
+        ):
+        return value

```

Comment:
İf 'in başlangıç ve bitiş satırlarını da kullanalım,2 satır azaltabiliriz burayı.

Reply:
Hatta sondaki type == none 'ı da bi üst satıra alabiliriz.